architecture/core_concepts/templates.adoc

```diff
@@ -87,12 +87,14 @@ application.
 
 Templates can be processed from a definition in a file or
 from an existing OpenShift API object.  Cluster administrators can
-ifdef::openshift-enterprise,openshift-origin[]
-link:../../install_config/install/first_steps.html#creating-instantapp-templates[define
-standard templates in the API]
+ifdef::openshift-origin[]
+link:../../install_config/imagestreams_templates.html#creating-instantapp-templates[define standard templates in the API]
+endif::[]
+ifdef::openshift-enterprise[]
+link:../../install_config/default_imagestreams_templates.html#creating-instantapp-templates[define standard templates in the API]
```

Comment:
I imagine any remaining references to `default_imagestreams_templates` are just leftovers/typos at this point and are meant to be `imagestreams_templates`, since Files Changed no longer shows `default_imagestreams_templates`.